File: PhaseGate/Java/DayOne/SimpleArithmeticApp.java
import java.util.Scanner;
import java.util.Random;

public class SimpleArithmeticApp {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        Random rand = new Random();

        int correctAnswers = 0;

        System.out.println("Welcome to Simple Arithmetic App");

        for (int i = 1; i <= 10; i++) {
		int num1 = rand.nextInt(50) + 1;             
		int num2 = rand.nextInt(num1) + 1; 
            int correct = num1 - num2;

            System.out.println(i + num1 + " - " + num2 + "=");
            System.out.print("Enter your answer: ");
            int userInput = input.nextInt();

            if (userInput == correct) {
		//correctAnswers += correct;
                //correctAnswers++;
            System.out.println(" Correct!");
            } else {
                System.out.println(" Incorrect! try next time");
            }
        }

        System.out.println("Na " + correctAnswers + " out of 10 you get.");
    }
}

File: PhaseGate/Python/day_one/simple_arithmetic_app.py
import random

print("Welcome to Simple Arithmetic App")
score = 0


for i in range(1, 2): 
    number_one = random.randint(0, 50)
    number_two = random.randint(0, 50)
    print(f"\n{number_one} - {number_two} =")  

    
    for attempt in range(1, 4): 
        
            answer = int(input("Answer: "))
            if answer == number_one - number_two: